js/simplicity.js

```diff
@@ -51,35 +51,4 @@ Drupal.behaviors.showMore.attach = function (context, settings) {
   });
 };
 
-Drupal.behaviors.selectOrOther = {};
-Drupal.behaviors.selectOrOther.attach = function(context, settings) {
-  // always show 'other' textfield for donation amount if select-or-other is enabled
-  var $donationComponent = $('.donation-amount', context);
-  var $donationSelect = $('.select-or-other-select', $donationComponent);
-  var $donationOtherRadio = $('[value="select_or_other"]', $donationSelect);
-  var $donationRegularRadios = $('input[type=radio]', $donationSelect).not($donationOtherRadio);
-  var $donationOther = $('.select-or-other-other', $donationComponent);
-  // hide by js so it will show up, when js is disabled
-  // therefor no functionality would be hidden
-  $donationOtherRadio.closest('.form-item').addClass('other-amount');
-
-  if ($donationOther.length > 0) {
-    // IE 7 needs bind on document for change
-    $(document).on('click', '.donation-amount .select-or-other-other', function (e) {
-      var value = e.target.value;
-      // validation should be done in another place
-      // trigger change to get picker to update the state
-      $donationOtherRadio.prop('checked', true).trigger('change');
-      $donationOther.addClass('select-or-other-checked');
```

Comment:
The new select-or-other js adds this class to `$('.select-or-other')` instead of `$('.select-or-other-other')` – this will break the styles, as [they target `input.select-or-other-checked`](https://github.com/moreonion/simplicity/blob/245db001451386bce8c01809214870b7076be453/css/form.less#L426)